### PR TITLE
Remove usages of RiscVOpensbiLib

### PR DESCRIPTION
**Pull request by JohnAZoidberg**
Counterpart of changab/edk2-staging-riscv#7

### CODE REVIEW

**Comment by changab** [comment]
Looks neat now. Please also create a branch for this change, as the counterpart of one OpenSBI library instance.

**Comment by JohnAZoidberg** [comment]
What do you mean?

What do we do about the functions we use not from `sbi/lib/` but from `sbi/utils/`, like: https://github.com/changab/edk2-platforms/blob/master/Platform/SiFive/U5SeriesPkg/Library/SerialIoLib/SerialPortLib.c#L51

I couldn't remove those. Should we split the utils part in a separate library in EDK2? It doesn't need to be initialized.

**Comment by changab** [comment]
> I couldn't remove those. Should we split the utils part in a separate library in EDK2? It doesn't need to be initialized.

Hmm, we probably don't need the initialization because of one opensbi instance, right?

File: Platform/SiFive/U5SeriesPkg/Universal/Dxe/TimerDxe/Timer.c [hunk 1]
```diff
@@ -9,10 +9,7 @@
 
 #include "Timer.h"
 #include <IndustryStandard/RiscVOpensbi.h>
```
Comment (JohnAZoidberg) [inline Platform/SiFive/U5SeriesPkg/Universal/Dxe/TimerDxe/Timer.c]: By including this, the `sbi_set_timer` calls happen through the SBI ecall. Therefore a dependency on `RiscVOpensbiLib` is not needed.

**Comment by JohnAZoidberg** [comment]
> Hmm, we probably don't need the initialization because of one opensbi instance, right?

The utils are independent of the library. They don't need to be initialized.

File: Platform/SiFive/U5SeriesPkg/Universal/Dxe/TimerDxe/Timer.c (hunk 1 above)
Comment (changab) [reply Platform/SiFive/U5SeriesPkg/Universal/Dxe/TimerDxe/Timer.c]: yes

**Comment by changab** [comment]
> > I couldn't remove those. Should we split the utils part in a separate library in EDK2? It doesn't need to be initialized.
> 
> Hmm, we probably don't need the initialization because of one opensbi instance, right?

SiFive UART in OpenSBI/Util is weird and should not be there in my opinion.
I think we can just copy the code in sifive-uart to SerialIoLib, those all belong to SiFive platform.

**Pull request closed without merging**